src/qtcore/qml/QMLPositioner.js

```diff
@@ -1,12 +1,12 @@
 function QMLPositioner(meta) {
     QMLItem.call(this, meta);
 
-    createProperty({ type: "int", object: this, name: "spacing" });
+    createProperty({ type: "int", object: this, name: "spacing", initalValue: 0 });
     this.spacingChanged.connect(this, this.layoutChildren);
     this.childrenChanged.connect(this, this.layoutChildren);
     this.childrenChanged.connect(this, QMLPositioner.slotChildrenChanged);
 
-    this.spacing = 0;
+    this.spacingChanged(0);
```

Comment:
Actually, forget my previous reaction : you're right, this.layoutChildren is more appropriate. I've fixed it.